README.md

```diff
@@ -182,7 +182,7 @@ This is the main Robot-Centric Elevation Mapping node. It uses the distance sens
 
 * **`min_update_rate`** (double, default: 2.0)
 
-    The mininum update rate (in Hz) at which the elevation map is updated either from new measurements or the robot pose estimates.
+    The mininum update rate (in Hz) at which the elevation map is updated either from new measurements or the robot pose estimates. If this rate is 0.0, the elevation map is only published with new measurements.
```

Comment:
To me it is not yet clear, is the map published upon receiving new measurements or as stated below in the ROS_INFO upon triggering the triggerFusion service?

Reply:
Is this clear with new commit?